.github/actions/build-container-image/action.yaml

```diff
@@ -60,6 +60,7 @@ runs:
 
     # https://github.com/docker/build-push-action
     - name: Build and push docker image to ghcr.io
+      if: github.event.pull_request.merged == true
```

Comment:
I understand the goal is to prevent pushing an image to the registry. However this condition would also prevent the build potentially hiding problems in the build. I would propose to remove this line and replace "push: true" in line 67 with something like "push: ${{ github.event.pull_request.merged == true }}"

Reply:
Thank you! Fixed, please check.